src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -2754,7 +2754,7 @@ private unsafe void SendManifest(byte[]? rawManifest)
 
         // Helper to deal with the fact that the type we are reflecting over might be loaded in the ReflectionOnly context.
         // When that is the case, we have the build the custom assemblies on a member by hand.
-        internal static Attribute? GetCustomAttributeHelper(
+        internal static bool IsCustomAttributeDefinedHelper(
             MemberInfo member,
 #if !ES_BUILD_STANDALONE
             [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.PublicProperties)]
```

Comment:
Is `DynamicallyAccessedMembers` still necessary now that we are no longer creating the attribute in this method?

See https://github.com/dotnet/designs/blob/main/accepted/2020/linking-libraries.md#requirements-on-type-values for more information on `DynamicallyAccessedMembers`.

Reply:
@josalem @sywhang

Reply:
Seems ok to remove